subsys/mgmt/osdp/src/osdp_common.c

```diff
@@ -74,3 +79,176 @@ struct osdp_cmd *osdp_cmd_get_last(struct osdp_pd *pd)
 {
 	return (struct osdp_cmd *)sys_slist_peek_tail(&pd->cmd.queue);
 }
+
+#ifdef CONFIG_OSDP_SC_ENABLED
+
+static inline int hex2int(char ch)
```

Comment:
There are already hex related functions like `hex2bin()` in `include/sys/util.h`, can't you use those?

Reply:
Thank you. I've switched to `hex2bin()` and removed my implementation.